deploy/kiali/kiali_cr.yaml

```diff
@@ -532,12 +550,18 @@ spec:
 #        type: "none"
 #        use_kiali_token: false
 #        username: ""
+#      component_status:
+#        app_label: ""
+#        is_core: false
```

Comment:
Again, these values commented here should be exactly what the true default is for documentation purposes.

```suggestion
#      component_status:
#        app_label: "prometheus"
#        is_core: true
```

Reply:
Good!